arviz/plots/densityplot.py

```diff
@@ -107,13 +108,20 @@ def plot_density(
     backend_kwargs: bool, optional
         These are kwargs specific to the backend being used. For additional documentation
         check the plotting method of the backend.
+        These are kwargs specific to the backend being used, passed to :func:`matplotlib.pyplot.subplots`
+         or  :func:`bokeh.plotting.figure`.
+
     show : bool, optional
         Call backend show function.
 
     Returns
     -------
     axes : matplotlib axes or bokeh figures
 
+    See Also
+    --------
+    plot_posterior :Plot Posterior densities in the style of John K. Kruschke’s book.
```

Comment:
Add space between `plot_posterior :` and `Plot` to pass the checks. Similarly, between `plot_density :` and `Generate`.